README.md

```diff
@@ -21,6 +21,16 @@ or if the GRAPHITEHOST/GRAPHITEPORT environment variables are found.
 
 Request data will be stored in memory (default) or in memcache.
 
+OSX Build Notes
+---------------
+Some additional steps may be needed to build carbonapi with cairo rendering on MacOSX.
+
+Install cairo:
+`$ brew install cairo --with-x11`
```

Comment:
before this command, I needed to run this command:

brew install Caskroom/cask/xquartz